meta/NotificationFactory.cpp

```diff
@@ -51,5 +52,8 @@ std::shared_ptr<Notification> NotificationFactory::deserialize(
     if (name == SAI_SWITCH_NOTIFICATION_NAME_TWAMP_SESSION_EVENT)
         return std::make_shared<NotificationTwampSessionEvent>(serializedNotification);
 
+    if (name == SAI_SWITCH_NOTIFICATION_NAME_TAM_TEL_TYPE_CONFIG_CHANGE)
+        return std::make_shared<NotificationTamTelTypeConfigChange>(serializedNotification);
+
```

Comment:
change description is stats st query but you also add tam ?

Reply:
Hi Kamil, this PR includes both Stats St Capability and TAM, the Stats St Capability is for TAM feature and the SAI was updated in a PR: https://github.com/opencomputeproject/SAI/pull/2089/files#diff-444febffce442e4277a1ced22d1ab94df045dfaa10a1b671ee35a7e5c2d9f683